packages/discord.js/src/structures/Webhook.js

```diff
@@ -140,7 +140,14 @@ class Webhook {
    * @property {Snowflake} [threadId] The id of the thread this message belongs to
    * <info>For interaction webhooks, this property is ignored</info>
    */
-
+  /**
+   * The channel that this webhook belongs to
+   * @type {?(GuildTextBasedChannel|ForumChannel)}
```

Comment:
`GuildTextBasedChannel` also includes thread channels - I am pretty sure webhooks cannot be created in thread channels. They are created on the parent channel and utilise a thread id parameter to send in a thread.